doc/Learn/HowTo-UseThemeService.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
@kazo0, @Xiaoy312 fyi This file should be moved to a proper folder and added to the proper toc.yml file
![image](https://github.com/unoplatform/uno.extensions/assets/16295702/388e05bf-17c3-4a92-ac95-19aaa608ff5b)

Reply:
@agneszitte are you able to help move this to correct location and then approve this PR?